packages/miniflare/test/plugins/d1/suite.ts

```diff
@@ -192,7 +192,7 @@ export default (binding: string, WORKER_PATH: string) => {
     const select = await db.prepare(`SELECT * FROM ${tableColours}`);
     let result: ColourRow | null = await select.first<ColourRow>();
     t.deepEqual(result, { id: 1, name: "red", rgb: 0xff0000 });
-    let id = await select.first<number>("id");
+    let id: number | null = await select.first<number>("id");
```

Comment:
When I remove it I get a type error. I think because `first` returns a `number` type when id is reassigned below it says can't assign `number | null` to a `number` type